<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Gross amount</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="dsbGross">
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Net amount</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbUpload">
        <property name="text">
         <string>&amp;Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="dsbNet">
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateEdit" name="deDay">
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="pbPdf">
     <property name="text">
      <string>&amp;Open...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>deDay</tabstop>
  <tabstop>dsbGross</tabstop>
  <tabstop>dsbNet</tabstop>
  <tabstop>pbPdf</tabstop>
  <tabstop>pbUpload</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
